Jenkinsfile_CNP

```diff
@@ -55,4 +58,11 @@ withPipeline(type, product, component) {
         steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/site/serenity/**/*'
         steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/tests/functional/**/*'
     }
+
+    //Temporary enable pact test at PR level
```

Comment:
why only temporarily? I mean, it would be even better if you can catch issues on the PR, wouldn't it?

Reply:
That's true I'll leave on for all Prs then